File: Foundation/Arrays/ceilroof.java
import java.util.Scanner;
public class ceilroof
{
    public static Scanner scn = new Scanner(System.in);
    public static void ceilroof(int[] arr, int num)
    {
        int n = arr.length;
        for(int i = 0; i < n; i++)
        {
            if(num > arr[i] && num < arr[i+1])
            {
                System.out.println(arr[i+1]);
                System.out.println(arr[i]);
            }
        }
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        int[] arr = new int[n];
        for(int i = 0; i < n; i++)
        {
            arr[i] = scn.nextInt();
        }
        int num = scn.nextInt();
        ceilroof(arr, num);
    }
}

File: Foundation/Arrays/ceilroof.cpp
#include<iostream>
#include<vector>
using namespace std;

void ceilroof(vector<int>& arr, int num)
{
    int n = arr.size();
    for(int i = 0; i < n; i++)
    {
        if(num > arr[i] && num < arr[i+1])
        {
            cout<<arr[i+1];
            cout<<arr[i];
        }
    }
}

int main()
{
    int n;
    cin >> n;
    vector<int> arr(n);
    for(int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    int num;
    cin >> num;
    ceilroof(arr, num);
}